java/raster-function-service/README.md

```diff
@@ -1,24 +1,43 @@
-# Raster Function Service
-Raster functions are operations performed on a raster to apply on-the-fly processing. This sample demonstrates how to create an image service raster and apply a hillshade raster function to it.
+# Raster function service
 
-![Raster Function Service App](raster-function-service.png)
+Load a raster from a service, then apply a function to it.
+
+![Image of apply raster function to raster from service](raster-function-service.png)
+
+## Use case
+
+Raster functions allow processing operations that can be applied to one or more rasters on the fly. Functions can be applied to rasters that come from a service. A land survey agency may apply hillshade and aspect functions to rasters with elevation data in order to better determine the topography of a landscape and to make further planning decisions.
 
 ## How to use the sample
-Click on the button to apply the Raster service.
+
+The raster function is applied automatically when the sample starts and the result is displayed.
 
 ## How it works
-* Create `ImageServiceRaster` using a `URL` and load it.
-* Create `RasterFunction` using a `JSON` string.
-* Get raster function's arguments with `RasterFunctionArguments`.
-* Set image service raster in the raster function arguments using `setRaster(rasterName, raster)`.
-* Create `Raster` using the `RasterFunction`.
-* Create `RasterLayer` using `Raster`.
-* Add `RasterLayer`  to the map operational layers with `map.getOperationalLayers()`.
+
+1. Create the `ImageServiceRaster` referring to the image server URL.
+2. Create the `RasterFunction` from a JSON string.
+3. Get the name of the raster argument to the function with `rasterFunction.getArguments().getRasterNames().get(0)`.
```

Comment:
This is the same case as the kotlin one; the workflow instead is to get the arguments and then get the names from it.